pycheribuild/projects/cross/nginx.py

```diff
@@ -128,3 +129,9 @@ def configure(self):
         self.configureEnvironment["NGX_OPENSSL_fett_path"] = str(BuildFettOpenSSL.get_instance(self).destdir) + openssl_dir
         self.configureEnvironment["NGX_OPENSSL_fett_rpath"] = openssl_dir + "/lib"
         super().configure()
+
+    def install(self):
+        super().install()
+        nginx_conf = str(self.destdir) + str(self._installPrefix) + "/conf/nginx.conf"
+        if os.path.exists(nginx_conf):
+            os.remove(nginx_conf)
```

Comment:
```suggestion
            self.deleteFile(nginx_conf)
```
So that it doesn't delete the file in --pretend mode.